var/spack/repos/builtin/packages/flamemaster/package.py

```diff
@@ -242,7 +242,7 @@ def cmake_args(self):
         args.append("-DEIGEN_INTEGRATION:BOOL=%s" % ("ON" if "+eigen" in spec else "OFF"))
         args.append("-DINSTALL_EIGEN:BOOL=%s" % ("ON" if "+eigen" in spec else "OFF"))
 
-        if "^amdlibflame" in spec:
+        if spec.satisfies("^amdlibflame"):
```

Comment:
```suggestion
        if spec.satisfies("^[virtuals=lapack] amdlibflame"):
```
@amd-toolchain-support Please verify this addition of explicit virtuals for `flamemaster ^amdlibflame`, per https://spack.readthedocs.io/en/latest/basic_usage.html#explicit-binding-of-virtual-dependencies.

Reply:
I updated it to use `virtuals`. In case, this can still be reverted